src/services/auth-services.ts

```diff
@@ -34,8 +39,9 @@ export default class AuthServices {
 
     const authApiUrl = sessionStorage.getItem(SessionStorageKeys.AuthApiUrl)
     const url = `${authApiUrl}entities/${businessId}`
+    const config = { headers: { 'Account-Id': store.getAccountId } }
 
-    return axios.get(url).then(response => {
+    return axios.get(url, config).then(response => {
```

Comment:
We pass in the account id and the API checks it against the business we're asking about. We'll get a HTTP 403 back if the business isn't affiliated to this account id.

NB - The API continues to accept "missing account id" and just returns the data if the business is affiliated with any account for the current login (Keycloak token).